frontend/src/app/core/setup/init-vendors.ts

```diff
@@ -31,36 +31,43 @@
 // dependencies required by classic (Rails) and Angular application.
 
 // Lodash
-require('expose-loader?_!lodash');
+import lodash from 'lodash';
 
 // jQuery
-require('expose-loader?jQuery!jquery');
-require('jquery-ujs');
-
-require('expose-loader?mousetrap!mousetrap/mousetrap.js');
+import 'jquery-ujs';
 
 // Angular dependencies
-require('expose-loader?dragula!dragula/dist/dragula.min.js');
-require('@uirouter/angular');
+import '@uirouter/angular';
 
 // Jquery UI
-require('jquery-ui/ui/position');
-require('jquery-ui/ui/disable-selection');
-require('jquery-ui/ui/widgets/sortable');
-require('jquery-ui/ui/widgets/dialog');
-require('jquery-ui/ui/widgets/tooltip');
+// import 'jquery-ui/ui/position';
+// import 'jquery-ui/ui/disable-selection';
+// import 'jquery-ui/ui/widgets/sortable';
+// import 'jquery-ui/ui/widgets/dialog';
+// import 'jquery-ui/ui/widgets/tooltip';
+import 'core-vendor/jquery-ui-1.14.1/jquery-ui';
```

Comment:
FYI I vendored all of the above dependencies via the jQuery UI [Download Builder](https://jqueryui.com/download/)